src/poetry_plugin_pypi_mirror/plugins.py

```diff
@@ -65,13 +65,10 @@ def activate(self, poetry: Poetry, io: IO):
             disable_cache=pypi_prioritized_repository.repository._disable_cache,
         )
 
-        priority = pypi_prioritized_repository.priority
-
         poetry.pool.remove_repository(DEFAULT_REPO_NAME)
         poetry.pool.add_repository(
             repository=replacement_repository,
-            default=priority == Priority.DEFAULT,
-            secondary=priority == Priority.SECONDARY,
+            priority=Priority.PRIMARY,
```

Comment:
Should this always be `Priority.PRIMARY`, or should we use whatever priority was already set for the PyPI repo (the removed code from line 68)?

For example, might there be some projects where PyPI has been set as a supplemental package source?

Reply:
I switched this to use whatever is set for the PyPI repo.